examples/sandbox/index.ts

```diff
@@ -958,7 +958,7 @@ $solanaTx.on("click", async (e) => {
   }
   if (core.supportsSolana(wallet)) {
     const res = await wallet.solanaSignTx({
-      addressNList: core.bip32ToAddressNList(`m/44'/501'/0'/0/0`),
+      addressNList: core.bip32ToAddressNList(`m/44'/501'/0'/0'`),
```

Comment:
standard 4 parts path - 5 path part is sad with Trezor
Paranoia tested with another hdwallet (Phantom) this doesn't break things:

<img width="442" height="548" alt="Image" src="https://github.com/user-attachments/assets/c5828c44-1fa6-42b5-ba39-7a26403316ba" />